geode-core/src/main/java/org/apache/geode/internal/cache/GemFireCacheImpl.java

```diff
@@ -1208,6 +1208,9 @@ private void initialize() {
           this.system.getConfig());
       initializeDeclarativeCache();
       completedCacheXml = true;
+    } catch (CacheXmlException e) {
```

Comment:
It looks like a number of exceptions can be thrown from initializeDeclarativeCache.
Would it be better to catch, log, and rethrow RuntimeException instead of CacheXmlException?